luajit: bump new version (#11235)
tarantool/tarantool: changelogs/unreleased/luajit-prof-fixes.md

## bugfix/tools

> Fix a bunch of bugs in the LuaJIT profilers:

Reviewer: ```suggestion
Fixed a bunch of bugs in LuaJIT profilers:
```

Author: Fixed, thanks!
```diff
diff --git a/changelogs/unreleased/luajit-prof-fixes.md b/changelogs/unreleased/luajit-prof-fixes.md
index 6cb20c318..feaf84140 100644
--- a/changelogs/unreleased/luajit-prof-fixes.md
+++ b/changelogs/unreleased/luajit-prof-fixes.md
@@ -1,7 +1,7 @@
 ## bugfix/tools
 
-Fix a bunch of bugs in the LuaJIT profilers:
-* `misc.sysprof.stop()` returns the correct error message if the profiler is not
+Fixed a bunch of bugs in LuaJIT profilers:
+* `misc.sysprof.stop()` returns a correct error message if the profiler is not
   running.
-* `misc.sysprof.start()` now raises the error if the given argument has an
-  incorrect type.
+* `misc.sysprof.start()` now raises an error if an argument has an incorrect
+  type.
```

> * `misc.sysprof.stop()` returns the correct error message if the profiler is not

Reviewer: ```suggestion
* `misc.sysprof.stop()` returns a correct error message if the profiler is not
```

Author: Fixed, thanks!
```diff
diff --git a/changelogs/unreleased/luajit-prof-fixes.md b/changelogs/unreleased/luajit-prof-fixes.md
index 6cb20c318..feaf84140 100644
--- a/changelogs/unreleased/luajit-prof-fixes.md
+++ b/changelogs/unreleased/luajit-prof-fixes.md
@@ -1,7 +1,7 @@
 ## bugfix/tools
 
-Fix a bunch of bugs in the LuaJIT profilers:
-* `misc.sysprof.stop()` returns the correct error message if the profiler is not
+Fixed a bunch of bugs in LuaJIT profilers:
+* `misc.sysprof.stop()` returns a correct error message if the profiler is not
   running.
-* `misc.sysprof.start()` now raises the error if the given argument has an
-  incorrect type.
+* `misc.sysprof.start()` now raises an error if an argument has an incorrect
+  type.
```

  running.

> * `misc.sysprof.start()` now raises the error if the given argument has an

Reviewer: ```suggestion
* `misc.sysprof.start()` now raises an error if an argument has an
```

Author: Fixed, thanks!
```diff
diff --git a/changelogs/unreleased/luajit-prof-fixes.md b/changelogs/unreleased/luajit-prof-fixes.md
index 6cb20c318..feaf84140 100644
--- a/changelogs/unreleased/luajit-prof-fixes.md
+++ b/changelogs/unreleased/luajit-prof-fixes.md
@@ -1,7 +1,7 @@
 ## bugfix/tools
 
-Fix a bunch of bugs in the LuaJIT profilers:
-* `misc.sysprof.stop()` returns the correct error message if the profiler is not
+Fixed a bunch of bugs in LuaJIT profilers:
+* `misc.sysprof.stop()` returns a correct error message if the profiler is not
   running.
-* `misc.sysprof.start()` now raises the error if the given argument has an
-  incorrect type.
+* `misc.sysprof.start()` now raises an error if an argument has an incorrect
+  type.
```
